src/Compilers/CSharp/Test/Semantic/Semantics/RecordStructTests.cs

```diff
@@ -1880,79 +1942,79 @@ bool M()
     }
 }
 ";
-            var comp = CreateEmptyCompilation(src, parseOptions: TestOptions.RegularPreview, references: new[] { corlibRef });
-            comp.VerifyEmitDiagnostics(
-                // (2,22): warning CS8907: Parameter 'X' is unread. Did you forget to use it to initialize the property with that name?
-                // record struct C(bool X)
-                Diagnostic(ErrorCode.WRN_UnreadRecordParameter, "X").WithArguments("X").WithLocation(2, 22)
-                );
-
-            Assert.Null(comp.GlobalNamespace.GetTypeMember("C").GetMember("X"));
-            var tree = comp.SyntaxTrees.Single();
-            var model = comp.GetSemanticModel(tree, ignoreAccessibility: false);
-            var x = tree.GetRoot().DescendantNodes().OfType<ReturnStatementSyntax>().Single().Expression;
-            Assert.Equal("System.Boolean System.ValueType.X { get; set; }", model.GetSymbolInfo(x!).Symbol.ToTestDisplayString());
-        }
+                var comp = CreateEmptyCompilation(src, parseOptions: TestOptions.RegularPreview, references: new[] { corlibRef });
+                comp.VerifyEmitDiagnostics(
+                    // (2,22): warning CS8907: Parameter 'X' is unread. Did you forget to use it to initialize the property with that name?
+                    // record struct C(bool X)
+                    Diagnostic(ErrorCode.WRN_UnreadRecordParameter, "X").WithArguments("X").WithLocation(2, 22)
+                    );
 
-        [Fact]
-        public void RecordProperties_PropertyInValueTuple_Static()
-        {
-            var corlib_cs = @"
-namespace System
-{
-    public class Object { }
-    public class ValueType
-    {
-        public static bool X { get; set; }
-    }
-    public class Attribute { }
-    public struct Void { }
-    public struct Boolean { }
-    public class Exception { }
-}
-";
-            var corlibRef = CreateEmptyCompilation(corlib_cs).EmitToImageReference();
+                Assert.Null(comp.GlobalNamespace.GetTypeMember("C").GetMember("X"));
+                var tree = comp.SyntaxTrees.Single();
+                var model = comp.GetSemanticModel(tree, ignoreAccessibility: false);
+                var x = tree.GetRoot().DescendantNodes().OfType<ReturnStatementSyntax>().Single().Expression;
+                Assert.Equal("System.Boolean System.ValueType.X { get; set; }", model.GetSymbolInfo(x!).Symbol.ToTestDisplayString());
+            }
 
-            var src = @"
-record struct C(bool X)
+            {
+                var src = @"
+readonly record struct C(bool X)
```

Comment:
Were adjacent tests combined here?

Reply:
Yes, they were sharing the corlib source

---
In reply to: [599778923](https://github.com/dotnet/roslyn/pull/51900#discussion_r599778923) [](ancestors = 599778923)